stabilizer/src/main/java/com/hazelcast/stabilizer/worker/selector/OperationSelector.java

```diff
@@ -0,0 +1,52 @@
+package com.hazelcast.stabilizer.worker.selector;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * Facility to select different operations based on probabilities. Each operations is represented as an enum item.
+ * <p/>
+ * Calling {@link #select()} method will select an operation according configured probabilities. It can return null if
+ * sum of configured probabilities is lower than 1. You can change this behaviour by configuring a default method.
+ *
+ * @param <T>
+ */
+public class OperationSelector<T extends Enum<T>> {
+    private final Random random;
+    private final Map<T, Integer> operations;
+    private final T defaultOperation;
+
+    OperationSelector(Map<T, Integer> operations, T defaultOperation) {
+        this.random = new Random();
+        this.operations = Collections.unmodifiableMap(operations);
+        this.defaultOperation = defaultOperation;
+    }
+
+    /**
+     * Select an operation according to configured probabilities.
+     * <p/>
+     * Selects a random operation, based on configured probabilities. If probabilities are smaller than 100 no operation may be
+     * selected. If a default operation is configured it is returned in that case. Otherwise <tt>null</tt> is returned.
```

Comment:
It's just the old behavior. I would change the `OperationSelectorBuilder` to throw an exception (instead of logging a warning), if the user missed to pass enough operations (fail fast). I think that's better than doing this during `OperationSelector.select()`.